python/hail/representation/genomeref.py

```diff
@@ -236,6 +236,27 @@ def from_file(cls, file):
         """
         return GenomeReference._from_java(Env.hail().variant.GenomeReference.fromFile(Env.hc()._jhc, file))
 
+    @handle_py4j
+    @typecheck_method(file=strlike)
+    def write(self, file):
+        """Write reference genome to file as JSON.
```

Comment:
Perhaps less curt? "Write this reference genome to a file in JSON format."

Are we going to say reference genome (which is standard) in docs, and keep the name GenomeReference? It really is a reference, not a genome, so I've come around to thinking the name GenomeReference makes sense and it won't cause confusion to keep as is.